src/core/Registry.ts

```diff
@@ -86,4 +91,22 @@ export class Registry<T> {
 			return fn(Identifier.parse(key), value, this)
 		})
 	}
+
+	public getTagRegistry() {
+		if (this.tags === undefined) {
+			this.tags = new Registry<HolderSet<T>>(new Identifier(this.key.namespace, `tags/${this.key.path}`))
+		}
+		return this.tags
+	}
+
+}
+
+export namespace Registry{
+	export const REGISTRY = new Registry<Registry<unknown>>(Identifier.create('root'))
+
+	export function register<T>(name: string, parser?: (obj: unknown) => T) {
```

Comment:
What's the reason to change this from static fields to a namespace?

Reply:
changed to static again field again